portality/migrate/reapplication/delete_unreapplied.py

```diff
@@ -0,0 +1,41 @@
+from esprit import tasks, raw
+from portality import models
+
+'''Delete journals and corresponding articles with incomplete reapplications'''
+
+# Connection to the ES index
+conn = raw.make_connection(None, 'localhost', 9200, 'doaj')
```

Comment:
The archiving is handled through esprit, which I usually set up thusly - is there a better way to dump them all to file by query?